File: src/gasing/division/division100.js
const a = 123456789;
const b = 100;

// GASING Division
let start = performance.now();
let hasil;
for (let i = 0; i < 10000000; i++) {
    hasil = parseInt(a.toString().slice(0, -2));
}
let gasing_time = (performance.now() - start) / 1000;
console.log("GASING Division 100:", gasing_time, "seconds");

// Computer Division
start = performance.now();
for (let i = 0; i < 10000000; i++) {
    hasil = Math.floor(a / b);
}
let computer_time = (performance.now() - start) / 1000;
console.log("General Division 100:", computer_time, "seconds");

console.log("\nGASING is", computer_time/gasing_time, "times faster");

File: src/gasing/alessandro_rumampuk/division/division_100.py
import time


a = 1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890
b = 100


# GASING Division
start = time.time()
for _ in range(10000000):
       hasil = str(a)[:-2]
end = time.time()
gasing_time = end - start
print("GASING Division 100:", gasing_time, "seconds")


# Computer Division
start = time.time()
for _ in range(10000000):
   hasil = a // b
end = time.time()
computer_time = end - start
print("General Division 100:", computer_time, "seconds")


print("\nGASING is", computer_time/gasing_time, "times faster")